### ISSUE TITLE
Is it possible to add a docker container for "Sons of the Forest"?

### ISSUE THREAD

**Issue by heusmich** [mschnitzer/ark-survival-ascended-linux-container-image#27]
Hi again,

Sorry to ask here in the ASA area, but I didn´t find a way to send you a private message or something similar.

My question is if you are also able to create a Docker container for "Sons of the Forest".
Since you created a very understandable install guide for the ASA container and it´s working very good, I was wondering if you could also create one for SotF.

Best regards
heusmich

**Comment by mschnitzer** [comment]
Hi,

while this would be possible to what I've seen, Sons of the Forest didn't catch my interest so far. I played it for a while with friends, but lost interest in playing it eventually.

As far as I can see, there are other projects out there that seem to provide what you are looking for already. If you look through google for a docker container for Sons of the Forest, it should show you some GitHub projects.

The reason I started this project for ASA, is because I was a fan of ASE and played it a lot and wanted to provide a server for me and my friends without paying any additional costs to a game server provider.

**Comment by heusmich** [comment]
Yes, I already found another project for this but the description how to get it working is not the best for me.
Since I don't have much knowledge in Docker, your description for ASA was very good for me, it described step by step what has to be done. And I got it working... :)
I also love ASE, so I'm happy that you published your project for ASA, so everbody could install and use it. Thanks again for this!
It's totally OK, if you don't want to publish a container for SotF, I just thought it doesn't hurt to ask... ;)
